cgi/oidc-callback.pl

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/perl -w
```

Comment:
This is still a really early PoC. My intention is to move this to `Routing.pm` and `Display.pm` before publishing the PR draft, but for the PoC stage, using `cgi` was a bit easier than figuring out the "best" `Routing.pm`. 🤷

Reply:
For now, I've decided not to move the files to ` Routing.pm` . That can still be done, if we decide to move all URLs to that structure. Right now, the required URL parameters seem to clash with the routing handling too much.